src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/Input/KeyConverter.cs

```diff
@@ -4,246 +4,326 @@
 
 //
 //
-// Description:
+// Description: KeyConverter - Converts a Key string
+// to the *Type* that the string represents and vice-versa
 //
-//      KeyConverter : Converts a key string to the *Type* that the string represents and vice-versa
-//
-// Features:
-//
-//
-//
-// 
 
-using System;
 using System.ComponentModel;    // for TypeConverter
 using System.Globalization;     // for CultureInfo
-using System.Reflection;
-using System.Windows;
-using System.Windows.Input;
-using System.Windows.Markup;
-using MS.Utility;
+
 using MS.Internal.WindowsBase;
 
 namespace System.Windows.Input
 {
     /// <summary>
-    /// Key Converter class for converting between a string and the Type of a Key
+    /// Converter class for converting between a <see langword="string"/> and <see cref="Key"/>.
     /// </summary>
-    /// <ExternalAPI/> 
     public class KeyConverter : TypeConverter
     {
-        /// <summary>
-        /// CanConvertFrom()
-        /// </summary>
-        /// <param name="context"></param>
-        /// <param name="sourceType"></param>
-        /// <returns></returns>
-        /// <ExternalAPI/> 
+        ///<summary>
+        /// Used to check whether we can convert a <see langword="string"/> into a <see cref="Key"/>.
+        ///</summary>
+        ///<param name="context">ITypeDescriptorContext</param>
+        ///<param name="sourceType">type to convert from</param>
+        ///<returns><see langword="true"/> if the given <paramref name="sourceType"/> can be converted from, <see langword="false"/> otherwise.</returns>
```

Comment:
nit: 
```suggestion
        ///<returns><see langword="true"/> if the given <paramref name="sourceType"/> can be converted, <see langword="false"/> otherwise.</returns>
```

Reply:
Fixed